src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] infoFromFile = readFromFile(fromFileName);
+        writeToFile(toFileName, createReport(infoFromFile));
+    }
+
+    public static String[] readFromFile(String fromFileName) {
+        File file = new File(fromFileName);
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            StringBuilder builder = new StringBuilder();
+            String value = reader.readLine();
+            while (value != null) {
+                builder.append(value).append(LINE_SEPARATOR);
+                value = reader.readLine();
+            }
+            return builder.toString().split(LINE_SEPARATOR);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't open file", e);
```

Comment:
The exception message 'Can't open file' is not entirely accurate, as the issue might not be with opening the file but reading from it. Consider using a more descriptive message like 'Error reading from file'.